app/i18n/en/index.ts

```diff
@@ -1143,6 +1143,7 @@ const en: BaseTranslation = {
     circlesGrowingKeepGoing:
       "Your circles grow as you welcome people to Blink – keep going!",
     points: "points",
+    innerCircleGrow: "Your inner circle grows when you send a Blink user their first sats!",
     calculatingYourCircles: "Calculating your circles...",
     innerCircleExplainer: "You're driving Bitcoin adoption with Blink, keep it up!",
     innerCircleExplainerCard: "I'm driving Bitcoin adoption with Blink!",
```

Comment:
Not related to this pr, but I think it would make sense to change  `peopleYouWelcomed: "people you welcomed"` to "peopleWelcomedByYou: "people welcomed by you"`. This is parallel to how we do it with `people welcomed by your circle".

Reply:
I agree with the discrepancy. These things should be identified much earlier at Figma imo.